include/libtorrent/announce_entry.hpp

```diff
@@ -187,18 +187,45 @@ namespace libtorrent {
 
 #ifndef TORRENT_NO_DEPRECATE
 		// deprecated in 1.2
-		// this is false the stats sent to this tracker will be 0
+		// all of these will be set to false or 0
+		// use the corresponding members in announce_endpoint
+		std::uint8_t fails:7;
```

Comment:
I introduced a macro recently to mark members as deprecated as well. would you mind adding it? like this:
``std::uint8_t TORRENT_DEPRECATED_MEMBER fails:7;``